src/frontend/fields/draft-actions.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <button
+    type="button"
+    :class="['action-button', drafts.showMetaBox ? 'active' : 'bg-white']"
+    @click.prevent="drafts.setShowMetaBox(!drafts.showMetaBox)"
+  >
+    <Icon name="info" class="action-icon" />
+  </button>
+  <button
+    v-if="shared.meta.hasAppPreview"
+    type="button"
+    :class="['action-button', drafts.showAppPreview ? 'active' : 'bg-white']"
+    @click.prevent="drafts.setShowAppPreview(!drafts.showAppPreview)"
+  >
+    <Icon name="mobile" class="action-icon" />
+  </button>
+  <button type="button" class="bg-white action-button" @click="emit('delete')">
```

Comment:
We will need a prop to hide the delete button on the preview page: 
```suggestion
  <button v-if="canDelete" type="button" class="bg-white action-button" @click="emit('delete')">
```